src/main/models/vehicles/Train.java

```diff
@@ -0,0 +1,48 @@
+package main.models.vehicles;
+
+import main.models.vehicles.interfaces.TrainLoading;
+
+public class Train implements TrainLoading {
```

Comment:
I would much prefer to see the regular subclassing of `SmallTrain `and `BigTrain` instead of providing constructor parameter of capacity.

Reply:
Subclassing added